corehq/ex-submodules/couchexport/tests/test_schema.py

```diff
@@ -110,7 +110,7 @@ def test_split_column_ignore_extras(self):
 
     def test_split_column_missing(self):
         self.post_it()
-        self._test_split_column([[None, None, None, None, None]])
+        self._test_split_column([[None, None, None, None, SCALAR_NEVER_WAS]])
```

Comment:
Hmm, that's not what I expected this to be, I thought it would become `[[SCALAR_NEVER_WAS]*5]`